docker/README.md

```diff
@@ -19,3 +19,40 @@ Build the container, for python users:
 After build finished, run the container:
 
     docker run --rm -it lightgbm
+
+## Using CLI via Docker
+
+Build a Docker image with LightGBM CLI:
+
+    docker build -t lightgbm-cli -f dockerfile-cli .
```

Comment:
Add these lines please
```
mkdir lightgbm-docker
cd lightgbm-docker
wget https://github.com/Microsoft/LightGBM/blob/master/docker/dockerfile-cli
```
and add code markdown (` ``` `)

Reply:
If it's not hard for you, please add the same lines to `Running the Python-package Container` section too
```
mkdir lightgbm-docker
cd lightgbm-docker
wget https://github.com/Microsoft/LightGBM/blob/master/docker/dockerfile-python
```

Reply:
Done, I assume you've meant the URLs to point to the raw content as well.

Reply:
There is no reason where to point raw or not, wget downloads successfully in both cases. But let it be raw, OK...